tinywallet/tinywallet/config.py

```diff
@@ -87,7 +87,7 @@ def __init__(self):
             print("**********************************************************")
         if args.loglevel:
             try:
-                if "," in args.loglevel or ":" in args.loglevel:
+                if any(ch in args.loglevel for ch in (",", ":")):
```

Comment:
I like my Martini DRY. :wink:

Reply:
Lame attempt at joke based on DRY: Don't Repeat Yourself. Sorry. :slightly_smiling_face:

Reply:
Ah! No, it's awesome. More jokes! I lol'd now that I got it!

Reply:
I discovered that among myriads of disciplines, just (tirelessly) following DRY alone it is possible to get very far in programming. Especially when you try hard to "dry up" code in unusual places.